packages/bot-web-ui/src/components/quick-strategy/quick-strategy.jsx

```diff
@@ -23,6 +23,15 @@ import { config } from '@deriv/bot-skeleton';
 import { popover_zindex } from 'Constants/z-indexes';
 import { connect } from 'Stores/connect';
 
+const getFieldValue = (list_items, value) => {
+    const dropdown_items = Array.isArray(list_items) ? list_items : [].concat(...Object.values(list_items));
+    const list_obj = dropdown_items.find(item =>
+        typeof item.value !== 'string' ? item.value === value : item.value?.toLowerCase() === value?.toLowerCase()
+    );
+
+    return list_obj?.text || '';
+};
+
```

Comment:
is it possible to use this function inside quick-strategy-store and return results as initial_values to the form?